### PR TITLE
refactor: line chart

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

* add plot
* divide types folder

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: types/store/store.d.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  axes: {
+    [key: string]: AxisData;
+  };
+  theme: Theme;
+  options: Options;
+  data: {
+    series: Series;
+    categories?: string[];
+  };
+  d: number; // @TODO: check where to use
```
Comment (shiren) [inline types/store/store.d.ts]: 반응형 테스트용 데이터에요. 만들어 질때마다  d를 업데이트합니다. 필요 없으는거임

File: src/brushes/axis.ts [hunk 2]
```diff
@@ -1,27 +1,8 @@
...
-export type LineModel = {
-  type: 'line';
-  x: number;
-  y: number;
-  x2: number;
-  y2: number;
-};
-
-export function tick(ctx: CanvasRenderingContext2D, { x, y, isYAxis }: TickModel) {
+import { TickModel, LabelModel, LineModel } from '../../types/components/axis';
```
[the remark this answers is not in the document]
Comment (jung-han) [reply src/brushes/axis.ts]: 넵!

File: types/store/store.d.ts (hunk 1 above)
Comment (jung-han) [reply types/store/store.d.ts]: 제거해도 괜찮은 값인가요?

File: src/component/axis.ts [hunk 3]
```diff
@@ -1,26 +1,22 @@
...
 
 type DrawModels = LabelModel | TickModel | LineModel;
-
-const isYAxis = (name: string) => name === 'yAxis' || name === 'yCenterAxis';
+type AxisModels = Record<string, DrawModels[]>;
+type CoordinateKey = 'x' | 'y';
 
 export default class Axis extends Component {
-  name!: axisType;
+  name!: AxisType;
```
[the remark this answers is not in the document]
Comment (jung-han) [reply src/component/axis.ts]: 넵

File: src/brushes/axis.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (jung-han) [reply src/brushes/axis.ts]: 메신저에서 이야기 나눈대로 `@t`로 수정하였습니다~

File: src/helpers/axis.ts [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export function isYAxis(name: 'yAxis' | 'xAxis' | 'yCenterAxis') {
```
[the remark this answers is not in the document]
Comment (jung-han) [reply src/helpers/axis.ts]: 이곳저곳 많이 쓰일줄알았는데 axis에서만 쓰이겠네요... 다시 원래대로 돌려놓도록 하겠습니다~

File: types/store/store.d.ts (hunk 1 above)
Comment (jung-han) [reply types/store/store.d.ts]: 일단 두겠습니다~ 이후에 알려주시면 삭제할게요

**Pull request merged**